hack/data-plane.sh

```diff
@@ -198,3 +200,16 @@ function data_plane_setup() {
   data_plane_build_push && k8s
   return $?
 }
+
+function data_plane_dispatcher_setup() {
```

Comment:
This function should be called in `hack/release.sh` to release the bundle.

Reply:
Yep - you right. This is IBM CE's preferred way of deploying. Adding..